### ISSUE TITLE
[bug] tauri info fails when @tauri-apps/api isn't in root package.json

### ISSUE THREAD

**Issue by betamos** [tauri-apps/tauri#4033]
### Describe the bug

`tauri info` panics when there's no `@tauri-apps/api` entry in the root package.json. See output below.

### Reproduction

1. Omit `@tauri-apps/api` from root package.json
2. Run `tauri info`

### Expected behavior

The `tauri info` command should not panic.

### Platform and versions

```shell
RUST_BACKTRACE=full npm run tauri info

> xxx-app@0.0.0 tauri
> tauri "info"


Environment
  › OS: Ubuntu 20.04 X64
  › Node.js: 16.15.0
  › npm: 8.5.5
  › pnpm: Not installed!
  › yarn: 1.22.10
  › rustup: 1.24.3
  › rustc: 1.60.0
  › cargo: 1.60.0
  › Rust toolchain: stable-x86_64-unknown-linux-gnu 

Packages
  › @tauri-apps/cli [NPM]: 1.0.0-rc.9
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error("unexpected end of input while parsing major version number")', src/info.rs:535:67
```
```


### Stack trace

_No response_

### Additional context

I use this dir structure:

- `./app/` (instead of src-tauri)
- `./ui/` (contains package.json with `@tauri-apps/api`
- `./package.json` root config, `@tauri-apps/cli` and shell scripts only

One reason I do this is because I mirror my UI to a standalone subtree repo.

**Comment by amrbashir** [comment]
fixed in #3996 , thanks for the report.

**Comment by betamos** [comment]
Doh! Another issue race. Thanks.
